ush/config.sh.RRFS_CONUS_3km_ens

```diff
@@ -75,46 +69,59 @@ ENDDAY=${DATE_LAST_CYCL:6:2}
 ENDHOUR="23"
 
 PREEXISTING_DIR_METHOD="upgrade" # "rename"
-INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 24:00:00"
+INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 12:00:00"
 BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 06:00:00"
-PROD_CYCLEDEF="${DATE_FIRST_CYCL}1900 ${DATE_LAST_CYCL}2300 01:00:00"
+PROD_CYCLEDEF="00 01-05,07-11,13-17,19-23 ${CYCLEDAY} ${CYCLEMONTH} ${STARTYEAR} *"
+PRODLONG_CYCLEDEF="${DATE_FIRST_CYCL}0000 ${DATE_LAST_CYCL}2300 06:00:00"
```

Comment:
For the DA cycling run, we also need 1-hr forecast at 00, 06, 12, 18 cycles to continue the cycling. So I would suggest PROD_CYCLEDEF to include all the hours except for the first spin-up cycle, if we are running ensemble free forecast in a separate workflow. In other words, keep the original PROD_CYCLEDEF setting and leave PRODLONG_CYCLEDEF out.

Reply:
Changed: PROD_CYCLEDEF is set to do hourly.